.github/workflows/ci.yaml

```diff
@@ -74,6 +75,28 @@ jobs:
               working-directory: "${{ steps.scaffold.outputs.dir }}"
               if: "${{ matrix.preset == 'js' }}"
 
+            - name: Rust smoke test
+              run: |
+                mkdir -p hello_world/src
+
+                cat >hello_world/BUILD.bazel <<EOF
+                load("@rules_rust//rust:defs.bzl", "rust_binary")
```

Comment:
hm, we don't want to ask developers to learn writing BUILD files, is there no gazelle extension for Rust at all currently?
We should consider adding one soon, could use our starlark extension host ("Orion") - doesn't have to be in this PR.

Reply:
I would love to write one!

Reply:
Yeah we would make significant dogfood usage in Silo too, fun project when you get some time.

I can haz approve here so I can land?